src/pykeen/checkpoints/keeper.py

```diff
@@ -120,5 +120,6 @@ def __call__(self, steps: Sequence[int]) -> Iterator[int]:
 
 #: a resolver for checkpoint keepers
 keeper_resolver: ClassResolver[CheckpointKeeper] = ClassResolver.from_subclasses(
-    CheckpointKeeper, default=CheckpointKeeper
+    CheckpointKeeper,  # type:ignore[type-abstract]
```

Comment:
Is that something that can be fixed upstream?

Reply:
I did a little looking into mypy and I don't think they support differentiating between abstract subclasses or concrete subclasses. I agree this is not a pretty workaround to use so many type ignores